lab_1/main.py

```diff
@@ -1,94 +1,121 @@
-"""
-Lab 1
-A concordance extraction
-"""
+import math
 
 
+def checker(con_object, type_ob):
+    if type_ob is list and type(con_object) is list:
+        len_ = len(con_object) > 0
+        bull_ = None not in con_object and bool not in con_object
+
+        if len_ and bull_:
+            return True
+    elif type_ob is str and type(con_object) is str and len(con_object) >= 0:
```

Comment:
String cannot have negative length.

Reply:
Yes, this is a typo. It was meant to be "!=0"